app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -119,6 +119,8 @@ class MainActivity : AppCompatActivity(),
                 UNKNOWN +
                 EMOJI +
                 LONG_TEXT
+
+        var testMode: Boolean = false
```

Comment:
You can detect the the test mode automatically, instead of having to update this variable manually:
```
private val isRunningTest : Boolean by lazy {
    try {
        Class.forName("android.support.test.espresso.Espresso")
        true
    } catch (e: ClassNotFoundException) {
        false
    }
}
```